.github/workflows/run_detekt_and_unit_tests.yml

```diff
@@ -72,6 +50,34 @@ jobs:
           name: CodeCoverageReportsTemplateCompose
           path: template-compose/app/build/reports/kover/
 
+      # template-xml
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: '11'
+
+      - name: Run Detekt on template-xml
+        working-directory: ./template-xml
+        run: ./gradlew detekt
+
+      - name: Archive Detekt reports on template-xml
+        uses: actions/upload-artifact@v2
```

Comment:
Same here, I think this can be update: actions/upload-artifact@v3

Reply:
Fixed in https://github.com/nimblehq/android-templates/pull/508/commits/799ed77f8e6da386ee2aead09e7a3681c793acd8